big_tests/tests/inbox_helper.erl

```diff
@@ -251,6 +255,40 @@ make_inbox_stanza(GetParams, Verify) ->
     GetIQ#xmlel{children = [QueryTag]}.
 
 
+-spec reset_inbox(
+        escalus:client(),
+        jid:jid() | escalus:client() | atom() | binary() | string())
+        -> ok.
+reset_inbox(From, To) ->
+        ResetStanza = make_reset_inbox_stanza(To),
+        escalus:send(From, ResetStanza),
+        Result = escalus:wait_for_stanza(From),
+        ?assert(escalus_pred:is_iq_result(ResetStanza, Result)),
+        ?assertNotEqual(undefined, exml_query:path(Result, [{element_with_ns, <<"reset">>,
+                                                             ?NS_ESL_INBOX_CONVERSATION}])).
+
+-spec make_reset_inbox_stanza(
+        undefined | jid:jid() | escalus:client() | atom() | string() | binary()) -> exml:element().
+make_reset_inbox_stanza(InterlocutorJid) when is_binary(InterlocutorJid) ->
```

Comment:
Are all these cases for `InterlocutorJid` types even used? I'm pretty sure that `escalus:client()`, `undefined` and `binary()` are sufficient.

Reply:
I though of leaving the application flexible so to not have to worry about what is being passed, but whatever, I'm deleting it.